src/redist/targets/packaging/osx/clisdk/Distribution-Template-x64

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
I wonder how hard it would be for this to leverage the task you just fixed in arcade?  Seems like this is pretty close to what the task would produce and might be more maintainable.

Reply:
Filed an issue to track doing this in the future: https://github.com/dotnet/installer/issues/12036